src/main/java/com/softserve/academy/spaced/repetition/repository/CardRepository.java

```diff
@@ -37,5 +39,14 @@
     List<Card> cardsQueueForLearningWithStatus(@Param("accountEmail") String accountEmail, @Param("deckId") long deckId,
                                                @Param("limitNumber") long limitNumber);
 
+    @Modifying
+    @Transactional
```

Comment:
Please move Transactional annotation to the service layer.

Reply:
Transactional annotation is now moved to the service layer.